demo/sticky-headers.html

```diff
@@ -1241,6 +1241,24 @@
 						</tbody>
 					</table>
 				</d2l-table-wrapper>
+
+				<p>One header and one data table</p>
+				<d2l-table-wrapper sticky-headers>
+					<table class="d2l-table">
+						<tbody>
+							<tr class="d2l-table-row-first" header>

```

Comment:
The classes should be added by the code, so isn't it unnecessary to specify them?